### PR TITLE
feat: square and colored props

### PR DESCRIPTION
**Pull request by alfinsyahruddin**
Hi @wdtamagi , I just added a new feature to make the Avatar component a square, and  randomly colored background if no image.
Hope you like it! 😉

### CODE REVIEW

**Comment by wdtamagi** [comment]
Hello @al-fin 
Thanks for your contribution, these features are nice to have :)
Just a minor comments for your PR.

File: src/Avatar.svelte [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
   export let name = 'Avatar';
   export let initials;
   export let src;
-  export let bgColor = 'lightGray';
+  export let bgColor;
   export let textColor = 'white';
   export let size = '50px';
   export let borderRadius = '50%';
+  export let square = false;
+  export let colored = false;
```
Comment (wdtamagi) [inline src/Avatar.svelte]: Maybe we can use a better name to this property, for indicate for the user that this property is related to a random color for background (eg. randomBgColor or something like that)

File: src/Avatar.svelte [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
+
+  const background = bgColor ? bgColor : colored ? getRandomColor() : 'lightGrey';
```
Comment (wdtamagi) [inline src/Avatar.svelte]: We can simplify this using the default value in the `bgColor` declaration, and for control the background we can just use a simple ternary like this:

`const background = colored ? getRandomColor() : bgColor;`

**Review by wdtamagi — COMMENTED** [review]
Just minor stuffs.

File: src/Avatar.svelte (hunk 1 above)
Comment (alfinsyahruddin) [reply src/Avatar.svelte]: okay, I'll revise it 👌

File: src/Avatar.svelte (hunk 2 above)
Comment (alfinsyahruddin) [reply src/Avatar.svelte]: good idea, thanks i'll revise it 👍

**Comment by alfinsyahruddin** [comment]
@wdtamagi, Please check my recent commit, I've fixed it 😁

**Review by wdtamagi — APPROVED** [review]
Very nice @al-fin Thanks for the PR.

**Pull request merged**